docs/csharp/language-reference/operators/snippets/shared/DelegateOperator.cs

```diff
@@ -51,5 +52,13 @@ private static void Discards()
             Console.WriteLine(constant(3, 4));  // output: 42
             // </SnippetDiscards>
         }
+
+        private static void Static()
+        {
+            // <SnippetStatic>
+            Func<int, int, int> sum = static delegate (int a, int b) { return a + b; };
```

Comment:
Do we have any code examples of variables being captured, and details as to why it is concerning? I realize that the `static` prevents this, but it might be more impactful to show some examples of capturing vs non-capturing with `static`.

Reply:
The example above is about capturing, but those two may be hard to compare and link.

Reply:
Sorry, I've missed the code file. I've meant the section in the lambda expression article:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/lambda-expressions#capture-of-outer-variables-and-variable-scope-in-lambda-expressions

Reply:
I also add an example when we added static local functions in C# 8: https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#static-local-functions

Interestingly, the `static` keyword page doesn't mention either static local functions, or static lambda expressions.

That should probably be a new PR.